models.py

```diff
@@ -22,8 +22,8 @@ class HadithCollection(db.Model):
     def serialize(self):
         return {
             "name": self.name,
-            "hasBooks": self.hasbooks == "yes",
-            "hasChapters": self.haschapters == "yes",
+            "hasBooks": self.hasbooksBoolean,
```

Comment:
Can we use a name without referring to type?

Some alternatives: includesBooks, containsBooks, booksExist

Reply:
Done